Apromore-Custom-Plugins/CSVImporter-Logic/src/main/java/org/apromore/service/csvimporter/services/MetaDataServiceCSVImpl.java

```diff
@@ -21,55 +21,51 @@
  */
 package org.apromore.service.csvimporter.services;
 
-import static org.apromore.service.csvimporter.utilities.CSVUtilities.getMaxOccurringChar;
+import com.google.common.base.Splitter;
+import com.opencsv.CSVReader;
+import org.apache.commons.io.input.ReaderInputStream;
+import org.apromore.commons.utils.Delimiter;
+import org.apromore.service.csvimporter.io.CSVFileReader;
+import org.apromore.service.csvimporter.model.LogMetaData;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.io.Reader;
+import java.io.*;
```

Comment:
At some time in the future when Checkstyle is enabled, I know that one of its rules is to avoid wildcards in import statements.  So we'll save a bit of time later on by not introducing them now.